src/server/hset_family.cc

```diff
@@ -579,6 +579,16 @@ OpResult<vector<string>> OpGetAll(const OpArgs& op_args, string_view key, uint8_
     }
   }
 
+  // Empty hashmaps must be deleted, this case only triggers for expired values
+  // and the enconding is guranteed to be a DenseSet since we only support expiring
```

Comment:
guranteed -> guaranteed

Reply:
Nice and I also think it's time to add a spellcheck to my vim config :)